src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -119,6 +122,49 @@ private void replaceKey(String oldKey, String newKey, Bundle extras, Bundle newE
     }
 
     /*
+     * Normalize localization for key
+     */
+    private String localizeKey(Context context, String key, String value) {
+        if (key.equals(TITLE) || key.equals(MESSAGE) || key.equals(SUMMARY_TEXT)) {
+            try {
+                JSONObject localeObject = new JSONObject(value);
```

Comment:
That minimzes the chances of a miss-catch, but it still exists. If we don't want to deal with the exceptions and can't use
```xml
<groovy-json-on-android-dream> a diff json lib </groovy-json-on-android-dream>
```
then this would be the the closest to safety, I guess.